docroot/themes/humsci/humsci_basic/humsci_basic.info.yml

```diff
@@ -15,3 +15,8 @@ regions:
   content: Content
   footer: Footer
   page_bottom: 'Page bottom'
+
+component-libraries:
+  humsci_basic:
```

Comment:
looks like .gitkeep files got added in this commit FYI

Reply:
I restructured this entire folder so we no longer having empty folders like this.